vultr/resource_startup_script.go

```diff
@@ -67,6 +70,12 @@ func resourceStartupScriptRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error getting startup script (%s): %v", d.Id(), err)
 	}
 
+	if (lib.StartupScript{}) == script {
+		log.Printf("[WARN] Removing Startup Script (%s) because it is gone", d.Id())
```

Comment:
Nit: could you fix the casing of “Startup Script” to be lowercase to match the other log statements like https://github.com/squat/terraform-provider-vultr/blob/master/vultr/resource_startup_script.go#L97